contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -742,7 +742,7 @@ BEGIN
     from
         sys.spt_tablecollations_view s_tcv
     where
-        s_tcv.object_id = sys.object_id(@object)
+        s_tcv.object_id = (SELECT sys.object_id(@object))
```

Comment:
I'm curious.  Why was this change necessary?

Reply:
Per Stefan: 

A root cause of the slow response time is [BABEL-1427](https://jira.rds.a2z.com/browse/BABEL-1427). This Jira puts further emphasis on evaluating volatility classes for system functions, in this case OBJECT_ID().

Performance can be improved by a workaround, altering the implementation of sys.sp_tablecollations_100 by putting the predicate with OBJECT_ID() in a subquery.

Replace:
        s_tcv.object_id = sys.object_id(@object)
With:
        s_tcv.object_id = ( SELECT sys.object_id(@object) )
This improves response time down to <10 seconds.